tests/console/commands/test_show.py

```diff
@@ -1041,10 +1041,7 @@ def test_show_outdated_local_dependencies(
 
     tester.execute("--outdated")
 
-    expected = """\
-cachy              0.2.0                       0.3.0
-project-with-setup 0.1.1 ../project_with_setup 0.1.2 ../project_with_setup
-"""
+    expected = """cachy 0.2.0 0.3.0 Cachy package"""
```

Comment:
This one is interesting. I guess if path dependencies are "special" in the sense that we always update them, we don't show them? I'm going to check what happens with VCS dependencies

Reply:
This looks like a regression to me. Probably, we still want to show them.

Reply:
Do we want all path dependencies to always show up as outdated? For me updated/not-updated for path dependencies is more "undefined" than "yes/no".

Reply:
Alright I think I was able to address this!